app/database_etl/etl_main.py

```diff
@@ -136,6 +136,9 @@ def main():
         # Make sure that filter options are still valid
         check_filter_options(dashboard_source)
 
+        # Update ordering and translations for filter options 
+        ingest_sample_frame_goi_filter_options()
```

Comment:
this function returns something but we aren't using the return info - let's remove what the function returns if we aren't gonna use it (i.e. just put return instead of return records)

Reply:
comment still has not been addressed i don't think @annoyingpr

Reply:
`ingest_sample_frame_goi_options` doesn't return anything anymore - you can see in airtable_records_getter.py:127!